atrium/models/AccountNumber.php

```diff
@@ -0,0 +1,43 @@
+<?php
+class AccountNumber {
```

Comment:
Is `AccountNumber` the right name for this class? Seems like Account maybe more appropriate since it isn't a number.